hathor/cli/run_node_args.py

```diff
@@ -0,0 +1,72 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import List, Optional
+
+from pydantic import Extra
+
+from hathor.utils.pydantic import BaseModel
+
+
+class RunNodeArgs(BaseModel, extra=Extra.allow):
```

Comment:
I understand we want to allow extra arguments for some commands. But how can we enforce that there is no unknown arguments?

Reply:
Thanks for the explanation.